AEPEdge.xcodeproj/project.pbxproj

```diff
@@ -1554,7 +1568,7 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
-				MARKETING_VERSION = "1.0.0-beta-1";
+				MARKETING_VERSION = "1.0.0-beta-2";
```

Comment:
I know we decided to change the version format to something like`1.0.0-beta.2` and we adopted this standard in the core extensions, we should update the Edge dev too before releasing.